### PR TITLE
Dockerise Mock CIR

### PR DESCRIPTION
**Pull request by katie-gardner**
### What is the context of this PR?
This PR has updated the docker files for runner to spin up the Mock CIR alongside the other supporting services to allow functional testing with eQ.

The docker files follow the same pattern as the addition of the mock SDS, and a minor change made to the functional testing to allow launching a schema by cir_instrument_id instead of schema name. 

### How to review
Check the new functional test passes and that if you use `make-dev-compose-up` and then launch runner, you are able to launch the CIR Schemas with the services running in docker. Make sure the new readme commands all work as expected

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by katie-gardner** [comment]
[Add dockerised CIR into eQ to allow functional testing (2 SP)](https://trello.com/c/mISHibIP/5869-add-dockerised-cir-into-eq-to-allow-functional-testing-2-sp)

File: docker-compose-dev-mac.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ services:
             SURVEY_RUNNER_URL: http://localhost:5000
             SURVEY_RUNNER_SCHEMA_URL: http://docker.for.mac.host.internal:5000
             SDS_API_BASE_URL: http://docker.for.mac.host.internal:5003
+            CIR_API_BASE_URL: http://host.docker.internal:5004
```
Comment (petechd) [inline docker-compose-dev-mac.yml:24]: Do we want to use `http://docker.for.mac.host.internal` here since it's the mac one?

**Review by petechd — APPROVED** [review]
Functionally works as expected, just one minor comment.

File: docker-compose-dev-mac.yml (hunk 1 above)
Comment (katie-gardner) [reply docker-compose-dev-mac.yml]: Oops, good spot, yep 👍

**Review by VirajP1002 — COMMENTED** [review]
Functionally tested and tests passed. With launcher then CIR schema dropdown is populated and using `/dump/debug` the `cir_instrument_id` appeared correctly.

**Review by berroar — APPROVED** [review]

**Pull request merged**